artwork/Modules/Project/Services/ProjectService.php

```diff
@@ -42,6 +42,32 @@ public function getProjectByCostCenter(string $costCenter): Project|null
         return $this->projectRepository->getProjectByCostCenter($costCenter);
     }
 
+    public function getProjects(): \Illuminate\Database\Eloquent\Builder
+    {
+        return $this->projectRepository->getProjectQuery([
```

Comment:
Das war etwas anders gemeint. Hier erwarte ich eigentlich nur ein call der z.B. $repository->getPaginatedProjects($orderBy, $start, $limit) heißt. Also der komplette Parameter der hier übergeben wird muss auch noch in das Repository.

Das Repository ruft dann auch paginate auf dem Query auf und du bekommst eine Collection. Mit der Collection kannst du bei Bedarf dann im Service weiterarbeiten.